src/DetailsView/components/details-view-overlay/settings-panel/settings/save-assessment-dialog/save-assessment-dialog-settings.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It would be good to add the new file to the strict null checks config. I think you can actually add an include for the whole `"src/DetailsView/components/details-view-overlay/settings-panel/**/*"` directory tree (or all of `details-view-overlay/**/*` with 2 minor fixes to `details-view-overlay.tsx`)

Reply:
Thank you! Done in c0a03da14d9e55c3bd79f909942b33c7992e69ac